index.html

```diff
@@ -60,10 +63,34 @@ <h2>Why it’s important to use natural and organic products on your skin</h2>
             our earth. So please take care of yourself and our planet, make a good decision and wear natural makeup products.
           </p>
         </div>
-    </div>
 
+        <div class="view-products hidden">
+          <div class="product-type-title">
+          </div>
+        <div class="view-products-row">
+      </div>
 
-  </div>
+      </div>
+        <div class="back-button hidden">
+          <button class="back">Back</button>
+        </div>
+      </div>
+
+    </div>
+
+    <div class="modal-search hidden">
+      <div class="pop-up">
+        <div class="dropdown-list">
+            <label for="product-type" class="label-product-type">Product Type:</label>
+            <select id="product-type" name="product-type">
+              <option value="product-type"></option>
+            </select>
```

Comment:
I'd suggest adding some text to the default `<option>` (like "Select a product type...").

---

![image](https://user-images.githubusercontent.com/16143771/138383376-d72f3094-aad7-421d-b0df-b0471a3138ba.png)